bowling-ball/Game.cs

```diff
@@ -1,19 +1,62 @@
-﻿using System;
+using System;
 
 namespace BowlingBall
 {
     public class Game
     {
-        public void Roll(int pins)
-        {
-            throw new NotImplementedException();
-        }
+        Frames [] frames = new Frames[10];

```

Comment:
naming convention? Please avoid repeated code review comments. Your  code should be consistent. Please do have a self review before you raise a PR.